### DISCUSSION TITLE
No confirmation/licence key after purchase

### DISCUSSION THREAD

**Discussion by The70sWoodsman** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#532] [Q&A] [answered]
I bought the Pro version this morning, but no confirmation email or licence key arrived and the support contact details are apparently in the email, which I didn't get. Does anyone know how to get in touch with the Al Dente team to solve this?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @The70sWoodsman,

I am Matthias from AppHouseKitchen.
Please send us a short informal email to. I will forward the confirmation email.

Cheers,
Matthias

**Reply by The70sWoodsman** [reply]
Awesome, Matthias. Thank you.

**Reply by MatthiasKerbl** [reply]
You had a small typo in your email address. I have forwarded the confirmation email to the correct email address now.
